### DISCUSSION TITLE
Unique run_every value per elastalert rule

### DISCUSSION THREAD

**Discussion by caoweixiong** [jertel/elastalert2#1094] [Q&A] [answered]
Is there a solution to the [elastalert#2836](https://github.com/Yelp/elastalert/issues/2836) problem in elastalert2?

I am using ES 7.14 and elastalert2-2.9.0, but it doesn't support.

Is there something that i am missing. I need to enable the unique value of run_every property in each rule such that it is considered. Is this feature already available per rule file? If not what must be done.

**Comment by jertel** [comment, marked as answer]
The `run_every` setting serves multiple purposes. If you're asking about rule execution scheduling, then yes, `run_every` is supported on a per-rule basis.
